lib/pages/profile/Profile.dart

```diff
@@ -82,7 +82,11 @@ class _ProfileState extends State<Profile> {
           {
             'profileTile':
                 ProfileTile("Rate us on the app store", CustomIcons.ic_rateus),
-            'function': () {StoreRedirect.redirect();}
+            'link':
+              Platform.isIOS 
+                ? "https://apps.apple.com/us/app/now-u/id1516126639"
```

Comment:
Instead of '/us/' it should be '/gb/' if the main audience is located in the UK. So that the app store forwards users to the right country's App Store app listing

Reply:
Great spot! Sorry, I didn't see this before! Ill change this now